confgenerator/windows-default-config.yaml

```diff
@@ -12,36 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
```

Comment:
This file is now identical to `default-config.yaml` (except for an extra comment line, which I assume is accidental). Do we still need both?

Reply:
The two files have been merged.